test/test-framework/deploy/crds/cache.example.com_memcachedrs_crd.yaml

```diff
@@ -19,12 +19,12 @@ spec:
         apiVersion:
           description: 'APIVersion defines the versioned schema of this representation
             of an object. Servers should convert recognized schemas to the latest
-            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
The updated links are invalid because our kubernetes tag for `k8s.io/apimachinery` contains old comments. I may have to add a regexp for these until we bump to v1.16.